cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -3,9 +3,20 @@
 set -e
 
 export CLUSTER_NAME="sriov"
-export KIND_NODE_IMAGE="kindest/node:v1.17.0"
+export K8S_VERSION="${K8S_VERSION:-1.17.0}"
 
-source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+function set_kind_params() {
+    local k8s_version=$1
+
+    if [ "$k8s_version" == "1.17.0" ]; then
+        export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.17.0}"
+        export KIND_VERSION="${KIND_VERSION:-0.7.0}"
+        export KUBECTL_PATH="${KUBECTL_PATH:-/kind/bin/kubectl}"
+    else
+        echo "Unsupported k8s version $k8s_version"
+        exit 1
```

Comment:
So how a custom version will work?

Reply:
you just set in the terminal before running cluster-up
it guards about the official supported `k8s_version`s

For example to run Kind 0.9.0
```
        export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600}"
        export KIND_VERSION="${KIND_VERSION:-0.9.0}"
        export KUBECTL_PATH="${KUBECTL_PATH:-/usr/bin/kubectl}"
```

Reply:
I do not think we will ever have more than one supported (default) version. Therefore, having a super-version looses its value.

This means that setting just the `KIND_NODE_IMAGE`, `KIND_VERSION` and `KUBECTL_PATH` defaults should be enough.